testsuite/integration/basic/src/test/java/org/wildfly/test/integration/jsp/JspCompilerTestCase.java

```diff
@@ -62,6 +63,6 @@ public static WebArchive deploy() {
 
     @Test
     public void test(@ArquillianResource URL url) throws Exception {
-        HttpRequest.get(url + "index.jsp", 10, TimeUnit.SECONDS);
+        HttpRequest.get(url + "index.jsp", TimeoutUtil.adjust(15), TimeUnit.SECONDS);
```

Comment:
This one shouldn't need the security manager permission because the test is @RunAsClient so the property isn't read in the server process.